README.md

```diff
@@ -45,11 +47,11 @@ instance.
 
 _dicom_parser_ provides _dict_-like access to the parsed values of the
 [header](https://dcm4che.atlassian.net/wiki/spaces/d2/pages/1835038/A+Very+Basic+DICOM+Introduction)'s
-data-elements. The raw values as read by _[pydicom]_ remain accessible through the _raw_ attribute.
+data-elements. The raw values as read by _[pydicom][]_ remain accessible through the _raw_ attribute.
```

Comment:
Codacy has a fix for the issue: [Spaces inside emphasis markers](https://app.codacy.com/gh/ZviBaratz/dicom_parser/pullRequest?prid=7629233)
```suggestion
data-elements. The raw values as read by_[pydicom][]_ remain accessible through the _raw_ attribute.
```